specs/chugsplash-manager.md

```diff
@@ -122,9 +130,10 @@ const getProxyByName = (
 * Bundles MUST be in the `APPROVED` state before they can be cancelled.
 * Cancelling a bundle MUST put the bundle in the `CANCELLED` state.
 * Cancelling a bundle MUST remove the active bundle.
-* If an executor has been selected within the last `executionLockTime` seconds (i.e. `block.timestamp` <= `timeClaimed + executionLockTime`):
-  * The ChugSplashManager MUST increase the `debt` owed to the current executor by `BUNDLE_EXECUTION_BOND_AMOUNT`.
-  * The ChugSplashManager MUST increase the `totalDebt` by the `BUNDLE_EXECUTION_BOND_AMOUNT`.
+* If an executor has been selected within the last `EXECUTION_LOCK_TIME` seconds (i.e. `block.timestamp` <= `timeClaimed + EXECUTION_LOCK_TIME`):
+  * The ChugSplashManager MUST increase the `debt` owed to the current executor by `OWNER_BOND_AMOUNT + EXECUTOR_BOND_AMOUNT`.
```

Comment:
Unrelated to the new action type.

We must add the `EXECUTOR_BOND_AMOUNT` to the `debt` here so that the executor can withdraw the full amount owed to them.